showcase/app/utils/component-states.ts

```diff
@@ -0,0 +1,2 @@
+// array of interaction states for presentation purposes in the showcase
+export const INTERACTION_STATES = ['default', 'active', 'hover', 'focus'];
```

Comment:
Not all the components use the same interaction states: https://github.com/search?q=repo%3Ahashicorp%2Fdesign-system+content%3A%2F%28%3F-i%29STATES+%3D+%2F&type=code

In that case, what do we do?

Reply:
I think we could export another const that doesnt have active and decide as a team how we handle disabled as that is the other "state" that is inconsistent across showcase pages.

Sometimes we show disabled states and other times we dont. In accordion, there are no examples of disabled accordions. Is a disabled accordion allowed? What about an accordion that has `containsInteractive=true`?

Reply:
Adding a few different variants for different included states here makes sense to me, as Lee suggested. I'm not sure how we would name the other variants though.

Reply:
Maybe you could do something like this:
```
export const INTERACTION_STATES_ALL = ['default', 'active', 'hover', 'focus', 'disabled'];
export const INTERACTION_STATES_NO_ACTIVE = ['default', 'hover', 'focus'];
export const INTERACTION_STATES_NO_DISABLED = ['default', 'active', 'hover', 'focus'];
```

As long as the list doesn't grow crazy-long I don't think it looks _too_ bad...